### DISCUSSION TITLE
Runtime GLTF import example

### DISCUSSION THREAD

**Discussion by ROBYER1** [needle-tools/needle-engine-support#67] [Demo and Starter Ideas]
Example of importing a GLTF file at runtime for a product viewer/internal 3d team asset sharing and previewing

**Comment by marwie** [comment]
You can do that already if you add for example a ``DropListener`` to your scene and then drag-drop glb files in your website

As a backend you should be able to use e.g. your glitch server.

Will add a sample for it.

**Reply by marwie** [reply]
There is also some code in the modules repository where we started implementing a google drive integration
https://github.com/needle-tools/needle-engine-modules
